### ISSUE TITLE
The questions about DWD datasets

### ISSUE THREAD

**Issue by startracker0** [WoojuLee24/OA-DG#6]
A heartfelt thank you to all contributors who have contributed to advancing domain generalization techniques on the DWD dataset.

I think https://github.com/WoojuLee24/OA-DG/blob/main/configs/OA-DG/dwd/faster_rcnn_r101_dc5_1x_dwd.py is the baseline of dwd, but I'm curious about whether the learning rate is set to 0.01 or 0.001? And also I also find the lr and milestones are differ from the settings for CDSD according to https://github.com/AmingWu/Single-DGOD#training. And I am curious about the code of oamix and oadg in dwd.

Thank you very much for answering my question, and I look forward to your reply.

**Comment by dazory** [comment]
We followed the setup in "[Single-Domain Generalized Object Detection in Urban Scene via Cyclic-Disentangled Self-Distillation](https://openaccess.thecvf.com/content/CVPR2022/papers/Wu_Single-Domain_Generalized_Object_Detection_in_Urban_Scene_via_Cyclic-Disentangled_Self-Distillation_CVPR_2022_paper.pdf)" (See [4. Experiments]-[4.1.Experimental Setup]-[Implementation Details]). According to the paper, the author set the learning rate to 0.001 for the experiment.

Also, you can ignore the commented out 0.01 in the configuration file. This is just a comment.

**Comment by startracker0** [comment]
Thanks a lot. I also wonder how much gpus do you use? 2 gpus?

**Comment by startracker0** [comment]
Thank you for your response. I have successfully reproduced your baseline of 48.1 with 4 3090s
